src/test/java/core/basesyntax/FruitShopTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax;
+
+import core.basesyntax.fileservice.LocalFileReader;
+import core.basesyntax.fileservice.LocalFileReaderImpl;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitShopTest {
+    private static LocalFileReader FILE_READER = new LocalFileReaderImpl();
+    private static final String INPUT_FIRST_FILE = "src/test/resources test1.csv";
+    private static final String INPUT_SECOND_FILE = "src/test/resources test2.csv";
+    private static final String INPUT_THIRD_FILE = "src/test/resources test3.csv";
+    private static final String INPUT_NONEXISTENT_FILE = "src/test/resources test4.csv";
+    private static final String REPORT_FIRST_FILE = "src/test/resources report_test1.csv";
+    private static final String REPORT_SECOND_FILE = "src/test/resources report_test2.csv";
+    private static final String REPORT_THIRD_FILE = "src/test/resources report_test3.csv";
+    private FruitShop fruitShop;
+
+    // Reads report and deletes it after reading
+    public String readResult(String reportPath) {
+        String[] path = reportPath.split(" ");
+        List<List<String>> listOfLists = FILE_READER.readFile(path);
+        String result = listOfLists.stream()
+                .map(s -> String.join(",", s))
+                .collect(Collectors.joining("\n"));
+        try {
+            Files.deleteIfExists(Paths.get(path[0] + FileSystems.getDefault().getSeparator() + path[1]));
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
if you are ignoring exception in test class - your test class implementation is wrong